src/data/baseline-data.js

```diff
@@ -0,0 +1,724 @@
+	["marker", 10],
+	["marker-end", 10],
+	["marker-mid", 10],
+	["marker-start", 10],
+	["r", 10],
+	["rx", 10],
+	["ry", 10],
+	["shape-rendering", 10],
+	["stop-color", 10],
+	["stop-opacity", 10],
+	["stroke", 10],
+	["stroke-color", 10],
+	["stroke-dasharray", 10],
+	["stroke-dashoffset", 10],
+	["stroke-linecap", 10],
+	["stroke-linejoin", 10],
+	["stroke-miterlimit", 10],
+	["stroke-width", 10],
+	["text-anchor", 10],
+	["text-rendering", 10],
+	["transform-origin", 10],
+	["vector-effect", 10],
+	["word-spacing", 10],
+	["x", 10],
+	["y", 10],
+	["color-interpolation-filters", 10],
+	["flood-color", 10],
+	["flood-opacity", 10],
+	["lighting-color", 10],
+	["tab-size", 10],
+	["border-collapse", 10],
+	["border-spacing", 10],
+	["caption-side", 10],
+	["empty-cells", 10],
+	["table-layout", 10],
+	["text-align", 10],
+	["text-align-last", 5],
+	["text-box", 0],
+	["text-box-edge", 0],
+	["text-box-trim", 0],
+	["text-combine-upright", 10],
+	["text-decoration", 10],
+	["text-decoration-color", 10],
+	["text-decoration-line", 10],
+	["text-decoration-skip", 10],
+	["text-decoration-skip-ink", 10],
+	["text-decoration-style", 10],
+	["text-decoration-thickness", 10],
+	["text-emphasis", 10],
+	["text-emphasis-color", 10],
+	["text-emphasis-position", 10],
+	["text-emphasis-style", 10],
+	["text-indent", 10],
+	["text-justify", 0],
+	["text-orientation", 10],
+	["text-shadow", 10],
+	["text-size-adjust", 0],
+	["text-spacing-trim", 0],
+	["-webkit-text-fill-color", 10],
+	["-webkit-text-stroke", 10],
+	["-webkit-text-stroke-color", 10],
+	["-webkit-text-stroke-width", 10],
+	["text-underline-offset", 10],
+	["text-underline-position", 10],
+	["text-wrap", 5],
+	["text-wrap-mode", 5],
+	["text-wrap-style", 0],
+	["touch-action", 10],
+	["transform-box", 5],
+	["transform", 10],
+	["backface-visibility", 10],
+	["perspective", 10],
+	["perspective-origin", 10],
+	["transform-style", 10],
+	["transition-behavior", 5],
+	["transition", 10],
+	["transition-delay", 10],
+	["transition-duration", 10],
+	["transition-property", 10],
+	["transition-timing-function", 10],
+	["user-select", 0],
+	["vertical-align", 10],
+	["writing-mode", 10],
+	["view-transition-class", 0],
+	["view-transition-name", 0],
+	["visibility", 10],
+	["white-space", 10],
+	["white-space-collapse", 5],
+	["orphans", 0],
+	["widows", 0],
+	["will-change", 10],
+	["word-break", 10],
+	["z-index", 10],
+	["zoom", 5],
+]);
+export const atRules = new Map([
+	["position-try", 0],
+	["keyframes", 10],
+	["import", 10],
+	["layer", 10],
+	["charset", 10],
+	["media", 10],
+	["font-face", 10],
+	["container", 5],
+	["counter-style", 5],
+	["view-transition", 0],
+	["media.display-mode", 0],
+	["font-face.src", 10],
+	["font-palette-values", 5],
+	["font-feature-values", 5],
+	["namespace", 10],
+	["page", 0],
+	["page.size", 0],
+	["media.prefers-color-scheme", 10],
+	["property", 5],
+	["scope", 0],
+	["starting-style", 0],
+	["supports", 10],
+]);
+export const types = new Map([
```

Comment:
I noticed you're not testing types yet, so this is more of a note for the future.

Types are going to be tricky to detect from the key. Some will be fairly straightforward- `abs`, `sign`, `anchor`, for instance, will all show up as functions with that name, in a value in an AST.

A few items like `gradient.*` and `easing-function.*` would show similarly show up as functions if you took the substring after the `.`. 

Others like `time` or `string` would not show up literally, but would need to be detected in a different way.

Reply:
Yeah, I figured for the first version of the rule I'd punt on this, but I wanted the data here for easy reference later.

The only way I can think to make this work is to maintain a separate dictionary that maps types to their expected AST locations.